frontend/src/home/Home.tsx

```diff
@@ -420,42 +270,60 @@ class HomeComponent extends React.Component<Props> {
             </HomePlan>
           </HomeTopInnerContainer>
         </HomeTop>
-        <OuterContainer>
-          <DragDropContext
-            onDragEnd={this.onDragEnd}
-            onDragUpdate={this.onDragUpdate}
-          >
-            <SidebarContainer>
-              <Sidebar />
-            </SidebarContainer>
-            <LeftScroll className="hide-scrollbar">
-              <Container>
-                <HomeAboveSchedule>
-                  <HomePlan>
-                    <h2>Plan Of Study</h2>
-                  </HomePlan>
-                  <HomeButtons>
-                    <PlanContainer>
-                      <PlanPopperButton
-                        variant="contained"
-                        onClick={this.updatePlan.bind(this)}
-                      >
-                        Update Plan
-                      </PlanPopperButton>
-                    </PlanContainer>
-                    <PlanContainer>
-                      <AddPlan />
-                    </PlanContainer>
-                    <SwitchPlanPopper />
-                  </HomeButtons>
-                </HomeAboveSchedule>
-                {this.renderYears()}
-                {this.renderTransfer()}
-              </Container>
-            </LeftScroll>
-          </DragDropContext>
-        </OuterContainer>
+        <EditableSchedule sidebarPresent={true} transferCreditPresent={true}>
+          {this.renderPlanHeader()}
+        </EditableSchedule>
       </>
+      // <>
```

Comment:
Can we delete these comments or is it best to save them (we always have git history if we need to revert)?

Reply:
Nah you're def right, I left this accidently when trying to resolve some merge conflicts